postcss.config.js

```diff
@@ -1,12 +0,0 @@
-module.exports = {
```

Comment:
This file was added when we webpacker-ified Argo, and we longer use webpacker for assets.